CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Not released
 
+## 0.12.0
```

Comment:
Should be 0.11.2 as it's a bugfix so just bump the patch ver

Reply:
Done.